assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -94,6 +61,19 @@ function LibraryProvider({ children }) {
     return EmptyPane;
   }, []);
 
+  const { highlightedTab } = useHighlights(({ tab: highlightedTab }) => ({
+    highlightedTab,
+  }));
+
+  useEffect(() => {
+    if (highlightedTab && LIBRARY_TAB_IDS.has(highlightedTab)) {
```

Comment:
do we need to check that `highlightedTab` exists since we're already checking if its in the set?
```
if (LIBRARY_TAB_IDS.has(highlightedTab)) {
```

Reply:
Lmao this also found a sneaky bug. Thanks for the catch!